docker/multi/Dockerfile

```diff
@@ -8,6 +8,8 @@ ENV TARGETPLATFORM=${TARGETPLATFORM}
 COPY fonts/* /usr/share/fonts/truetype/
 COPY src src/
 
+RUN apt-get install -y -qq libwebp-dev
+
```

Comment:
Could we move this to down below where we do the "bulk" install command? Same with the webkit docker

Reply:
Sure thing. I wanted to keep it separated to keep the font installation in its own step